.github/workflows/check-link.yml

```diff
@@ -0,0 +1,18 @@
+name: Check URL Validity
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run monthly
+    - cron: "0 0 1 * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: lychee Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.0.6
+      - name: Fail if there were link errors
+        run: exit ${{ steps.lychee.outputs.exit_code }}
```

Comment:
Maybe log errors in an issue: https://github.com/NixOS/nixpkgs/blob/693a938bef1447c7ef3f988b5b5b7363018db862/.github/workflows/merge-staging.yml#L32

Reply:
Definitely doable: https://github.com/lycheeverse/lychee-action#receiving-issues-containing-the-error-report.

Should it do this in addition to failing the check, or in place of failing the check?

Reply:
whatever is easier.

Reply:
Alright, fixed in 4ddce16

Reply:
...and replaced with a comment generator instead, so that issues don't get overloaded if someone produces a bunch of bad prs or something.